Character.cpp

```diff
@@ -98,20 +98,28 @@ void Character::attackInternal(Character& other)
             b) your stats are boosted 10%
             c) the initial value of your stats is updated to reflect this boosted stat for the next time you defeat another character.
       */
-        assert(false);
+        hitPoints = (*initialHitPoints * 1.1);
+        armor = (*initialArmorLevel * 1.1);
+        attackDamage = (*initialAttackDamage * 1.1);
+        
+        *initialHitPoints = hitPoints;
+        *initialArmorLevel = armor;
+        *initialAttackDamage = attackDamage;
+
         std::cout << getName() << " defeated " << other.getName() << " and leveled up!" << std::endl;        
     }
 }
 
 void Character::printStats()
 {
     std::cout << getName() << "'s stats: " << std::endl;
-    assert(false);
+    //assert(false);
     /*
     make your getStats() use a function from the Utility.h
     */
-    std::cout << getStats(); 
-    
+    //std::cout << getStats(); 
+    std::cout << getCharacterStats(this);
```

Comment:
you did not need to modify this function.  call `getStats();` as before
- [ ] revise